package.json

```diff
@@ -21,25 +21,44 @@
     "compile": "tsc -watch -p ./",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "node ./node_modules/vscode/bin/test",
-    "build-server": "./node_modules/.bin/gulp build_server"
```

Comment:
"./node_modules/.bin/gulp build_server" is more portable

Reply:
I didn't realise I removed that, ill put it back.